projects/GKv3/GKComponents/GKUI/Components/HyperView.cs

```diff
@@ -236,9 +236,9 @@ private void ArrangeText()
                     fTextSize = new ExtSize(xMax + 2 * fBorderWidth, yPos + 2 * fBorderWidth);
                     SetImageSize(fTextSize);
                 } finally {
-                    fAcceptFontChange = true;
+                    ResumeLayout();
 
-                    //ResumeLayout();
+                    fAcceptFontChange = true;

```

Comment:
fAcceptFontChange should be set after ResumeLayout layout is finished. Othewise it will endlessly try resize the view.

Reply:
👍